CommonTools/PileupAlgos/python/Puppi_cff.py

```diff
@@ -31,6 +31,9 @@
                        NumOfPUVtxsForCharged = 2,
                        PtMaxCharged   = 20.,
                        PtMaxNeutralsStartSlope = 20.,
+                       useVertexAssociation = True,
+                       vertexAssociationQuality = cms.int32(6),
+                       vertexAssociation = cms.InputTag('primaryVertexAssociationJME','original'),
                        #candName      = cms.string('packedPFCandidates'),
```

Comment:
Since you are there, you can remove the type specification also from `candName` and `vertexName`, even if commented out